app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_results = {}
+
+    def wrapper(*args, **kwargs) -> None:
+        arg_key = (args, frozenset(kwargs.items()))
```

Comment:
i think its unnessesary, we can do just smthn like this 'if args in cache_dict:'
wbu?